.vscode/settings.json

```
@@ -10,5 +10,8 @@
   "githubPullRequests.ignoredPullRequestBranches": [
     "master"
   ],
-  "git.ignoreLimitWarning": true
+  "git.ignoreLimitWarning": true,
+  "git.branchProtection": [
+    "master"
```

The following branches are protected according to the GitHub settings:
```suggestion
    "master",
    "stable",
    "nightly-testing-*",
    "bump/v4*"
```
(does `*` work in VS Code as well to match everything?)

---

Let me check

---

Unfortunately, unlike the .gitignore style globs for directory names, it seems * does not work. I will remove those but keep master and stable, I don't necessarily think it's worth remembering to update the VSCode settings every time a nightly-testing- comes out